### ISSUE TITLE
Dragging image onto page fails, with bad path

### ISSUE THREAD

**Issue by jhpedemonte** [maqetta/maqetta#862]
Drag SampleBanner.jpg on to page.  Won't work as before, with error in console:

```
GET http://localhost:8081/maqetta/user/maqettaUser/ws/workspace/project1/project1/SampleBanner.jpg 404 (Not Found)
```

Notice the double instances of "project1".  The generated HTML source has:

``` html
<img src="./project1/SampleBanner.jpg"></img>
```

**Comment by jhpedemonte** [comment]
In **ImageDragSource.js**, `dragStart` has a call to `this.data.getPath()`.  `this.data` is a **File** resource, and `getPath()` returns "./project1/SampleBanner.jpg".  Is that right?

But the issue is that this is within a file that is a sibling of SampleBanner.jpg, so the correct value should just be "SampleBanner.jpg".  What is the correct way to ignore the "project1" dir?

**Comment by peller** [comment]
see #810

**Comment by jhpedemonte** [comment]
Thanks.  Closing as duplicate of #810.

**Comment by childsb** [comment]
closed 810, using this bug to track the DND bug.

**Comment by childsb** [comment]
Fixed with commit cfde584b97bfdcd8f288b6cd0152df9a06737245
